### ISSUE TITLE
Discoverability of this project

### ISSUE THREAD

**Issue by JWizards** [syncthing-gtk/syncthing-gtk#77]
I was wondering why this project isn't listed in the community contributions section of the official syncthing [docs](https://docs.syncthing.net/users/contrib.html#contributions)

 The older repo without maintainers is still linked there and can be misleading.

**Comment by acolomb** [comment]
Good point. Would you be willing to make a PR for this change at https://github.com/syncthing/docs?

**Comment by Rikiub** [comment]
I made a PR here https://github.com/syncthing/docs/pull/873

**Comment by maymage** [comment]
The links on the Flahub page are also a mess. 

https://github.com/flathub/me.kozec.syncthingtk/issues/190

**Comment by Salamandar** [comment]
Hi, to be perfectly clear, I created this fork with some other people that were annoyed at the original, dead project, in the hope that they would contribute too, but they didn't. I don't have the energy to maintain this alone, and as an embedded software dev I am not up to date on gnome software development.

So if anyone wants to step up (not alone but with me), feel free to ping me (@Salamandar:matrix.org)

**Comment by maymage** [comment]
@Salamandar 
I'm sorry, that I can't help you out myself -_-

Having read 
https://discourse.gnome.org/t/proposal-integrate-syncthing-into-gnome-settings/15387/4
perhaps it's a good idea to ask for co-maintainer ship-help on GNOME Discourse. Off-cloud syncing is often discussed in the GNOME sphere.

You may ping @tbernard over there, I know, he's on that topic. 

Aldo https://gitlab.gnome.org/bertob/app-ideas/-/issues/256
